src/Game/Managers/NameOverHeadManager.cs

```diff
@@ -95,22 +95,28 @@ public static bool IsAllowed(Entity serial)
 
         public static void Open()
         {
-            if (_gump != null)
+            if (_gump == null)
             {
-                return;
+                _gump = new NameOverHeadHandlerGump();
             }
 
-            _gump = new NameOverHeadHandlerGump();
+            if (!_gump.IsEnabled)
```

Comment:
These kind of checks are not necessary, just set the variable directly to `true` or `false`